console/templates/store.html

```diff
@@ -69,8 +70,8 @@
             <td class="table__cell">{{ item.metadata.user_id }}</td>
             <td class="table__cell">{{ item.submitted_at }}</td>
             <td class="table__cell">{{ item.invalid }}</td>
-            <input type="hidden" name="json_data" value="{{ item }}">
-            <td align="center">
+            <input id="json_data" type="hidden" name="json_data" value="{{ item }}">
```

Comment:
Why submit the whole json data when you only need the tx_id?

I'd be tempted to do something like this:
'''input type="hidden" id="tx_id" name="tx_id" value="{{item.tx_id}}"'''